.pre-commit-config.yaml

```diff
@@ -42,6 +42,9 @@ repos:
     rev: 1.7.2
     hooks:
     -   id: darker
-        args: [--isort]
-        additional_dependencies:
-          - isort~=5.9
+
+
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: 'v0.1.4'
```

Comment:
Probably not too relevant, but it would be nice if we use the same Ruff version for all repos to have some homogeneity in the code rules.

That said, I would bump up [the version of nucypher repo](https://github.com/nucypher/nucypher/blob/v7.0.0/.pre-commit-config.yaml#L44) instead of using a old version on this repo.

Reply:
Agreed. The `nucypher` repos is using an old version.